src/id5-api.js

```diff
@@ -28,12 +27,23 @@ ID5.init = function (options) {
     const referer = getRefererInfo();
     utils.logInfo(`ID5 detected referer is ${referer.referer}`);
 
-    const storedId = JSON.parse(utils.getCookie(cfg.cookieName));
-    const storedDate = new Date(+utils.getCookie(`${cfg.cookieName}_last`));
+    const storedResponse = JSON.parse(utils.getCookie(cfg.cookieName));
+    const storedDate = new Date(+utils.getCookie(lastCookieName(cfg)));
     const refreshNeeded = storedDate.getTime() > 0 && (Date.now() - storedDate.getTime() > cfg.refreshInSeconds * 1000);
-    if (storedId) {
-      ID5.userId = storedId.ID5ID;
-      utils.logInfo('ID5 User ID already available:', storedId, storedDate, refreshNeeded);
+    const expiresStr = (new Date(Date.now() + (cfg.cookieExpirationInSeconds * 1000))).toUTCString();
+    const nb = incrementAndGetNb(cfg, expiresStr);
```

Comment:
This nb is is the number of times API has been initialized. It might be diffrent from what we get from prebidjs, probably more "number of bid requests using our uid" or even "number of adaptor who requested ou uid". I would name this one `nbPage`, same in the /g/g endpoint call

Reply:
https://github.com/id5io/id5-api.js/pull/8/commits/029290c8c0ade11c37bd2631b867eed7a63b61f6

Reply:
fixed